library/std/src/sys/psp/os.rs

```diff
@@ -1,10 +1,13 @@
 use super::{unsupported, Void};
 use crate::error::Error as StdError;
-use crate::ffi::{OsStr, OsString};
+use crate::ffi::{CString, OsStr, OsString};
 use crate::fmt;
 use crate::io;
+use crate::os::psp::ffi::OsStrExt;
 use crate::path::{self, PathBuf};
 
+static mut CWD: Option<CString> = None;
```

Comment:
AFAIK there is no way to ask for cwd, so it's just stored here. PSPSDK does something similar.